### ISSUE TITLE
Many examples are broken in Chrome due to mixed content errors

### ISSUE THREAD

**Issue by mikepianka** [potree/potree#1297]
Many of the examples in the README are failing to load the pointcloud due to mixed content errors.
![image](https://user-images.githubusercontent.com/47948499/222161697-52651c2a-947b-44d8-8afe-7e57968362c3.png)

Also, there was a bug fix awhile ago that hasn't been incorporated in the examples that fixed an issue where a chrome update broke multi-point measurements.
https://potree.org/potree/examples/measurements.html

**Comment by m-schuetz** [comment]
Regarding the mixed content errors - does your browser automatically upgrade "http" links to "https"? Because some links are explicitly http precisely because the data is on a server that does not support https. 

Regarding the chrome update - I'm now focusing on Potree 2 so unfortunately I don't have the time to fix Potree 1.x anymore.

**Comment by mikepianka** [comment]
> Regarding the mixed content errors - does your browser automatically upgrade "http" links to "https"? Because some links are explicitly http precisely because the data is on a server that does not support https.
> 
> Regarding the chrome update - I'm now focusing on Potree 2 so unfortunately I don't have the time to fix Potree 1.x anymore.

Correct, Chrome is linking out via https automatically. I updated the title. I know the workaround but just wanted to bring it to your attention for newcomers to the repo that may not realize that and would just see a blank example.
